lib/index.d.ts

```diff
@@ -427,6 +427,7 @@ declare namespace Realm.Sync {
         partial?: boolean;
         fullSynchronization?: boolean;
         _disableQueryBasedSyncUrlChecks?:boolean;
+        custom_http_headers?: any;
```

Comment:
Shouldn't this be

```ts
{ [header: string]: string }
```

I don't suppose we can pass in a complex object.

Reply:
True, your suggestion is more accurate.